changelog.dd

```diff
@@ -18,6 +18,9 @@ $(BUGSTITLE Library Changes,
         has been deprecated.))
     $(LI $(RELATIVE_LINK2 has-function-attributes, Added `std.traits.hasFunctionAttributes`
         as a user-friendly way to query for function attributes.))
+    $(LI $(RELATIVE_LINK2 bitwise, Added `std.range.bitwise` to create
+        a bitwise adapter over an integral type range, consuming the range elements
+        bit by bit.))
```

Comment:
The `changelog.dd` wasn't removed yet, but it will be [soon](https://github.com/dlang/phobos/pull/5033).
Could you please create a file in the new `changelog` folder? This will also end merge conflicts with the changelog file.